src/__test__/TestApolloServer.ts

```diff
@@ -2,12 +2,19 @@ import { ApolloServerBase } from 'apollo-server-core'
 import * as fs from 'fs'
 import * as path from 'path'
 import * as depthLimit from 'graphql-depth-limit'
-import { getConfig } from '../config'
+import resolvers from '../resolvers'
+import { buildContext } from '../Context'
+
+/**
+ * A server compatible with apollo-server-testing
+ *
+ * Enables direct querying of the service, sans HTTP server.
+ *
+ * @packageDocumentation
+ */
 
 export async function TestApolloServer (): Promise<ApolloServerBase> {
-  process.env.HASURA_URI = 'http://localhost:8090/v1/graphql'
-  const { context, resolvers } = await getConfig()
-  // ApolloServer has express encapsulated, so the 'apollo-server-testing' package requires the base
+  const context = await buildContext('http://localhost:8090/v1/graphql')
```

Comment:
Left-over from the previous refactor, use of CI server to run the test meant it wasn't detected